docs/DEVELOPING.md

```diff
@@ -235,7 +235,7 @@ The following configuration fields need to be provided for setting up the connec
 
 ##### `service_account_credentials`
 
-The service account credentials generated from Google Cloud Storage.  Default value is [here](https://github.com/elastic/connectors-python/blob/main/connectors/sources/google_cloud_storage.py#L100)
+The service account credentials generated from Google Cloud Storage.  Default value is [here](https://github.com/elastic/connectors-python/blob/8.7/connectors/sources/google_cloud_storage.py#L103)
```

Comment:
This may point to a different line because you are using a branch not a specific commit.

Reply:
Good point. Maybe it's better to link to the file and mention the function name, where the user can <kbd>command</kbd> + <kbd>F</kbd> rather than linking to specific line numbers which will vary.

Reply:
Currently, I have removed the `google cloud storage` and `Azure Blob Storage` changes from this PR. We will raise follow up PR for all connectors